base_content_space.sol

```diff
@@ -455,40 +459,73 @@ BaseCtFactory20200928110000PO: Replace tx.origin with msg.sender in some cases
     function getContentObjectBin() public pure returns (bytes memory) {
-        return currentObjBin;
+        return type(BaseContent).creationCode;
     }
 }
 
 contract BaseContentFactory is Ownable {
 
-    bytes32 public version ="BaseCtFactory20200928110000PO"; //class name (max 16), date YYYYMMDD, time HHMMSS and Developer initials XX
+    bytes32 public version ="BaseCtFactory20230412230000GG"; //class name (max 16), date YYYYMMDD, time HHMMSS and Developer initials XX
+    bytes4 constant createContentSig = bytes4(keccak256("createContentInstance(address,address,address)"));
+
+    // TODO: *not* adding a setter here means that we need to create a new factory if we want to change the helper - i.e. change
+    //  the base content code. I *think* this is what we want to do.
+    address payable factoryHelper;
 
     constructor(address payable _spaceAddr, address payable _helperAddr) public {
         contentSpace = _spaceAddr;
         factoryHelper = _helperAddr;
     }
 
-    bytes4 constant createContentSig = bytes4(keccak256("createContentInstance(address,address,address)"));
-
-    // TODO: *not* adding a setter here means that we need to create a new factory if we want to change the helper - i.e. change
-    //  the base content code. I *think* this is what we want to do.
-    address payable factoryHelper;
+    // creationBytecode uses the creationCode of baseContent ..
+    // but this does not work and the contracts reverts
+    function creationBytecode(
```

Comment:
So what's the deal here? Does this now work or not? Or is the comment simply out-of-date? Very confusing...